qtranslate_options.php

```diff
@@ -55,10 +55,22 @@
 // Language code format: ISO 639-1 (2 alpha), 639-2 or 639-3 (3 alpha)
 define( 'QTX_LANG_CODE_FORMAT', '[a-z]{2,3}' );
 
+/**
+ * Option names.
+ */
+const QTX_OPTIONS_MODULES_STATE = 'qtranslate_modules_state';
+const QTX_OPTIONS_MODULE_ACF    = 'qtranslate_module_acf';
+const QTX_OPTIONS_MODULE_SLUGS  = 'qtranslate_module_slugs';
```

Comment:
These definitions would look better in modules source files.
It would be better to segregate code which depends on module list from core files.

Reply:
The idea is that all options definitions should be kept consistent in a unique place (the options names are used both for front and admin). Currently it's hard to to understand where a general option is set or read in the code because the values are spread everywhere and most of the settings are generated from intermediate strings (with  `qtranslate_` prefix). The modules should not be fundamentally different regarding the options interface. The other options should also converge to something more comprehensive.